src/randoop/FailureSet.java

```diff
@@ -5,17 +5,27 @@
 import java.util.List;
 import java.util.Set;
 
+import randoop.operation.MethodCall;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.MutableSequence;
+import randoop.sequence.MutableVariable;
+import randoop.sequence.Statement;
+import randoop.sequence.Variable;
+
 public class FailureSet {
   
   private Set<Failure> failures = new LinkedHashSet<Failure>();
   
   public static class Failure {
-    public final StatementKind st;
+    
+    public final Statement st;
     public final Class<?> viocls;
-    public Failure(StatementKind st, Class<?> viocls) {
-      this.st = st;
+    
+    public Failure(Statement st2, Class<?> viocls) {
```

Comment:
removed.  I will be more careful with Eclipse tools. Not sure where that came from.